src/ts/components/playbacktimelabel.ts

```diff
@@ -132,7 +134,15 @@ export class PlaybackTimeLabel extends Label<PlaybackTimeLabelConfig> {
       }
     };
 
+    let playbackReadyHandler = () => {
```

Comment:
Possible name. Also we need to call this function by default on initialization.
```suggestion
    let updatePlaybackTimeFormat = () => {
```